lib/instrumentation/index.js

```diff
@@ -25,12 +26,14 @@ Instrumentation.prototype.start = function () {
   var self = this
   this._started = true
 
-  this._queue = new Queue({flushInterval: this._agent._flushInterval}, function (err, transactions) {
-    if (err) {
-      debug('queue returned error: %s', err.message)
-    } else if (self._agent.active && transactions) {
-      request.transactions(self._agent, transactions)
-    }
+  this._queue = new Queue({flushInterval: this._agent._flushInterval}, function onFlush (transactions) {
```

Comment:
Not part of this PR, but I'm hoping in the future we also get `flush_size` means when the size is reached, it flushes.

Reply:
It's already done, but I need this PR to be merged first before I can commit it to not get merge conflicts 😄

Reply:
FYI: #56